package-res/resources/web/pentaho/visual/base/types/selectionModes.js

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define(function() {
+  "use strict";
+
+  /**
+   * The `selectionModes` namespace contains a collection of methods that represent different ways
+   * to combine the current selection filter with the selection filter created by the user interaction with a visualization.
+   *
+   * @name selectionModes
+   * @namespace
+   * @memberOf pentaho.visual.base.types
+   */
+  return /** @lends pentaho.visual.base.types.selectionModes */{
+
+    /**
+     * Replaces the current selection filter with the provided filter.
+     *
+     * @param {pentaho.data.filter.AbstractFilter} current - The filter representing the current selection filter.
+     * @param {pentaho.data.filter.AbstractFilter} input - The filter that will replace the current selection filter.
+     * @return {?pentaho.data.filter.AbstractFilter} The input filter.
+     * @static
+     */
+    REPLACE: function(current, input) {
+      return input;
+    },
+
+    /**
+     * Adds the input filter to the current selection filter, but
+     * previously selected items will be unselected.
+     *
+     * @param {pentaho.data.filter.AbstractFilter} current - The current selection filter.
+     * @param {pentaho.data.filter.AbstractFilter} input - The filter that will be added the current selection filter,
+     * removing all the items previously selected.
+     * @return {?pentaho.data.filter.AbstractFilter} The previous selection filter and the input that filter items combined,
```

Comment:
@swagner8412 there is a rogue "that" in there.
the original should be: The previous selection filter and the input filter items combined, but previously selected items will be unselected.

And with your changes (I think it would be like this) :
The previous selection filter and the input filters items combined. Previously selected items will be unselected.